tests/components/airzone_cloud/test_sensor.py

```diff
@@ -27,12 +27,30 @@ async def test_airzone_create_sensors(
     assert state.state == "-77"
 
     # Zones
+    state = hass.states.get("sensor.dormitorio_pm1")
+    assert state.state == "3"
+
+    state = hass.states.get("sensor.dormitorio_pm2_5")
+    assert state.state == "4"
+
+    state = hass.states.get("sensor.dormitorio_pm10")
+    assert state.state == "3"
+
```

Comment:
Could you also add a test for the AQI sensor you added? This seems to have been missed.

Reply:
> Could you also add a test for the AQI sensor you added? This seems to have been missed.

Sure, thanks for pointing this out!
Done in e03f30ec95682ddc18d95aa80392762d7d898480